src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -902,7 +902,7 @@ public NBTTagList writeToNBT(NBTTagList nbtTagListIn)
         {
             if (!(this.armorInventory.get(j)).isEmpty())
             {
-                NBTTagCompound nbttagcompound1 = new NBTTagCompound();
+                final NBTTagCompound nbttagcompound1 = new NBTTagCompound();
                 nbttagcompound1.setByte("Slot", (byte) (j + 100));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 100 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)